addons/mail/static/src/components/message/message.xml

```diff
@@ -49,7 +49,7 @@
                                 />
                             </t>
                         </div>
-                        <t t-if="message.isCurrentPartnerAuthor and threadViewer and threadViewer.thread">
+                        <t t-if="message.isCurrentPartnerAuthor and threadViewer and threadViewer.thread and threadViewer.thread.members.length &lt;= 2">
```

Comment:
This does not feel like a great condition to have. Data about seen shouldn't reach here if that case is not supposed to be managed. If data is in store, it's normal to display it.